tests/modules/sightings/resources/test_sighting_encounter_fields.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+
+from tests.modules.sightings.resources import utils as sighting_utils
+from tests.extensions.edm import utils as edm_utils
+from tests.modules.configuration.resources import utils as conf_utils
+import pytest
+import random
+import json
+
+from tests.utils import module_unavailable
+
+
+def _rnd_lat():
```

Comment:
I would say that these should not be private but should in fact be public... in test/utils.py

Reply:
moved to test/utils